### DISCUSSION TITLE
Fuukei Sakurairo 11

### DISCUSSION THREAD

**Discussion by mirai-mamori** [mirai-mamori/Sakurairo#347] [General]
### 让你更加接近你所爱的生活
Sakurairo 11 让用户与之所爱，更近一步，其设计旨在为用户在工作、家庭或学校等生活的方方面面提供支持。
从全新的“首页”和视频，到每一种交互、字体和图标，Sakurairo 11 都为用户带来了更加现代化、整洁美观，且令人耳目一新的体验。Sakurairo 11 的“首页”采用全新设计，可以让用户快速访问所需展示的内容。

同时，凭借云和 Maho API 的强大功能，用户可以在“首页”中获得统一的视觉体验，无论他们此前是在哪个设备上访问，即使是在一台 Android 或 iOS 设备上，“首页”依然能让用户倍感欣慰。

<hr /><em>This discussion was created from the release <a href='https://github.com/mirai-mamori/Sakurairo/releases/tag/2.2.11'>Fuukei Sakurairo 11 </a>.</em>

**Comment by vip503168815** [comment]
这个主题我上传的网站错误主题有问题
